src/Employer/Employer.Web/ViewModels/ErrorMessages.cs

```diff
@@ -3,7 +3,7 @@
     public class ErrorMessages
     {
         public const string VacancyCannotBeViewed = "The vacancy '{0}' can't be viewed.";
-        public const string VacancyNotAvailableForEditing = "The vacancy '{0}' is not available for editing.";
+        public const string VacancyNotAvailableForEditing = "The vacancy '{0}' has been edited by one of your colleagues. Your changes have not been saved.";
```

Comment:
I think this message makes sense for POSTs but not for GETs.  If this error is triggered during a GET I think the main culprit will be people clicking an old hyperlink/favourite.  Maybe we should keep the "The vacancy '{0}' is not available for editing." message for GETs?